panic in renderCallType

I'm debugging a scenario where we hit the panic in renderCallType:

The code flow is:

- Get new block
- Make VM from that block
- Apply all the transactions in that block sequentially (and do some logging/tracing etc stuff)
- Occasionally, we'll get a panic in renderCallType, downstream of vm.ApplyTx

```
github.com/lmittmann/w3/w3vm/hooks.renderCallType(0x4e?)
	2025-06-24 18:23:33.316	
goroutine 42456361 [running]:
	2025-06-24 18:23:33.316	
panic: unknown call type   
```

It has happened when applying these transactions:
- 0xdd83c41d8b2d6c940783cc24191541c9c80e3d428e7bf28df807cae663a33a4b
- 0x0f3baf732c924145dd894eed962897d3fbb1347e59f39c7122c6c825229b1ba9

Any ideas? I'm speculating this is a bug in the library, since we're just applying transactions that were included on-chain, but I'm not sure. Can I provide any more info to help?

--- a/w3vm/hooks/call_tracer.go
+++ b/w3vm/hooks/call_tracer.go
@@ -196,6 +196,8 @@ func renderCallType(typ byte) string {
 	switch vm.OpCode(typ) {
 	case vm.CALL:
 		return ""
+	case vm.CALLCODE:
+		return "callcode "
 	case vm.STATICCALL:
 		return stylesStaticcall.Render("static") + " "
 	case vm.DELEGATECALL:
@@ -204,8 +206,10 @@ func renderCallType(typ byte) string {
 		return "create "
 	case vm.CREATE2:
 		return "create2 "
+	case vm.SELFDESTRUCT:
+		return "selfdestruct "
 	default:
-		panic(fmt.Sprintf("unknown call type %92x", typ))
+		panic(fmt.Sprintf("unknown call type %02x", typ))
 	}
 }
 

